pkg/tsdb/azuremonitor/azuremonitor-datasource.go

```diff
@@ -248,6 +248,8 @@ func (e *AzureMonitorDatasource) parseResponse(amr AzureMonitorResponse, query *
 
 		frame := data.NewFrameOfFieldTypes("", len(series.Data), data.FieldTypeTime, data.FieldTypeNullableFloat64)
```

Comment:
> I wonder if it is safe to assume the first field is always the time field here?

It is safe because in this line we are setting the first field type as `data.FieldTypeTime`. For me it's a bit weird that there is no way to set the field name when creating the frame and we have to do it manually afterwards but that's what the sdk exposes. It may be worth an issue for that repo but this should be fine.